### ISSUE TITLE
PEP 541 Request: dashtools

### ISSUE THREAD

**Issue by andrew-hossack** [pypa/pypi-support#1950]
### Project to be claimed

`PROJECT_NAME`: https://pypi.org/project/dash-tools

### Your PyPI username

`USER_NAME`: https://pypi.org/user/andrew_hossack

### Reasons for the request

I am the owner of `dash-tools` (see [dash-tools GitHub](https://github.com/andrew-hossack/dash-tools) and [dash-tools on PyPi](https://pypi.org/project/dash-tools/)) and would like to change the project name to `dashtools` for easier type-ability as a command line tool. Thanks.

### Maintenance or replacement?

Replacement

### Source code repositories URLs

https://github.com/andrew-hossack/dash-tools

### Contact and additional research

E: andrew_hossack@outlook.com

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

**Comment by pbelskiy** [comment]
I have same problem, but with helix-swarm package, I think PyPI should ignore this error in naming if packages from same author.

**Comment by andrew-hossack** [comment]
> I have same problem, but with helix-swarm package, I think PyPI should ignore this error in naming if packages from same author.

Agreed @pbelskiy, and it seems like PEP 541 doesn't fit for this type of request either.

**Comment by yeraydiazdiaz** [comment]
Hi @andrew-hossack, yes, this is not technically a PEP 541 request but we haven't gone around to creating a specific template for these issues. In any case they're usually solved by a @pypa/warehouse-admins releasing the project name.
